src/Sharding/ShardingManager.ts

```diff
@@ -1,10 +1,12 @@
-import { Client, ClientOptions } from 'discord.js';
+import { Client, ClientOptions, Intents } from 'discord.js';
 import { MasterIPC } from '../IPC/MasterIPC';
 import { Cluster } from '../Cluster/Cluster';
 import { http, SharderEvents } from '../Util/Constants';
 import { EventEmitter } from 'events';
 import { cpus } from 'os';
-import { isMaster, setupMaster } from 'cluster';
+import cluster from 'cluster';
+const isPrimary = cluster.isPrimary
```

Comment:
please import isPrimary from cluster directly in the above import statement, no extra const needed